cmd/otelinmemexporter/store.go

```diff
@@ -155,37 +157,45 @@ func (s *Store) Add(ld pmetric.Metrics) {
 
 // GetAll returns all the aggregated values for all the configured
 // aggregation configs.
-func (s *Store) GetAll() map[string]float64 {
+func (s *Store) GetAll() map[string]map[string]float64 {
```

Comment:
Here and elsewhere: it might be worth altering the comments now that a map-of-maps is returned

Reply:
Done!